common/scripts/kind_provisioner.sh

```diff
@@ -165,6 +165,12 @@ function setup_kind_cluster() {
     echo "No existing kind cluster with name ${NAME}. Continue..."
   fi
 
+  # Create the docker network beforehand so "kind create cluster" doesn't create it with ipv6
```

Comment:
@yxun kind create cluster creates the kind cluster based on the `IP_FAMILY`. 
Can you explain why this change is required?